src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,249 @@
-import {Command} from 'clipanion'
+import * as fs from 'fs'
+import * as path from 'path'
+import util from 'util'
 
-import {renderLambdaFlareHeader} from './renderers/flare-renderer'
+import {LambdaClient, LambdaClientConfig} from '@aws-sdk/client-lambda'
+import {AwsCredentialIdentity} from '@aws-sdk/types'
+import axios from 'axios'
+import {BaseContext, Command} from 'clipanion'
+import FormData from 'form-data'
+import JSZip from 'jszip'
+
+import {API_KEY_ENV_VAR, AWS_DEFAULT_REGION_ENV_VAR, CI_API_KEY_ENV_VAR} from './constants'
+import {getAWSCredentials, getLambdaFunctionConfig, getRegion} from './functions/commons'
+import {requestAWSCredentials} from './prompt'
+import * as commonRenderer from './renderers/common-renderer'
+import * as flareRenderer from './renderers/flare-renderer'
+
+const {version} = require('../../../package.json')
+
+const ENDPOINT_URL = 'https://datad0g.com/api/ui/support/serverless/flare'
```

Comment:
do we want to hardcode a staging URL like this?

Reply:
Hey @juan-fernandez, we will hard code it for staging in the meantime. We will not merge this to the `main` branch until we have the right endpoint merged in production. 

We will bring another PR to update this to the user specific site endpoint in the future, wdyt?